sym_api_client_python/clients/sym_bot_client.py

```diff
@@ -51,6 +54,16 @@ def __init__(self, auth, config):
         self.async_agent_session = None
         self.bot_user_info = None
         self.health_check_client = None
+        #Required for aiohttp to use truststore
+        if self.config.data[_TRUSTSTORE_PATH]:
+            logging.debug("Setting truststorePath for async calls to {}".format(config.data[_TRUSTSTORE_PATH]))
+            self.async_ssl_context = ssl.create_default_context(cafile=self.config.data[_TRUSTSTORE_PATH])
+        else: 
+            logging.debug("Setting truststore for async calls to system truststore")
```

Comment:
since it will the default maybe we don't need the debug log

Reply:
Will remove.